test/extensions/transport_sockets/tls/ocsp/ocsp_test.cc

```diff
@@ -197,7 +218,7 @@ TEST_F(Asn1OcspUtilityTest, ParseResponseDataBadResponderIdVariantTest) {
       // version
       0xa0,
       0x02,
-      1,
+      0,
       0,
```

Comment:
Since this test doesn't concern itself with the version, I think you should remove the version entirely here (and change the 7 above to a 3 since you've reduced the size of the sequence.

Reply:
Sure.